gulp/formatBuilditJobPostings.js

```diff
@@ -23,11 +28,7 @@ module.exports = function formatBuilditJobPostings(allJobs) {
     }
 
     // Append this job's details to the jobLocation
-    jobLocation.jobs.push({
-      title: job.title,
-      type: job.typeOfEmployment,
-      url: job.url
-    });
+    jobLocation.jobs.push(job);
```

Comment:
Since this is a directly pushed reference to an object should it be a copy instead of a direct reference `{...job}`? https://stackoverflow.com/questions/39736397/is-this-a-good-way-to-clone-an-object-in-es6

Reply:
Good catch! I'll fix that right away.

Reply:
Aw! Scratch that.

The Cypress tests were (thankfully) failing after this change because the job posting links disappeared. The reason is that those `job` objects aren't simple JS objects, but instances of a class (defined within the `@buildit/job-postings` package we use). The `url` property is actually a getter method. So, when I clone that object the getter is lost (because it's not an enumerable property).

I'm therefore reverting this back to how it was. Hope that's OK.

Reply:
No worries, it's ideal to pass in a copy for immutability but okay if you can't.